src/admin/shared/components/ContentPicker/ContentPicker.tsx

```diff
@@ -98,7 +100,7 @@ export const ContentPicker: FunctionComponent<ContentPickerProps> = ({
 	// when selecting a type, reset `selectedItem` and retrieve new item options
 	useEffect(() => {
 		inflatePicker(null, setItemOptions);
-	}, [selectedType]); // eslint-disable-line
+	}, [selectedType, setItemOptions]); // eslint-disable-line
```

Comment:
selectedType moet hier nog weg
inflatePicker moet je nog toevoegen aan de dependency array en de functie in een useCallback wrappen

of lukte dat niet? 😕

Reply:
Had hem vergeten te committen en pushen bij de vorige merge... 🤦‍♂🤣 Bij deze zit hij dus in deze PR nu.